packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -342,6 +342,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'dropdown',
```

Comment:
Tree view items shouldn't be focusable. That might be a part of bigger issue. If yes feel free to move it to another issue.

Reply:
@asvae, but it's accessibility requirements. I'll add a property to toggle tree items focusable state.

Reply:
Yeah, they should be focusable. The only problem we've met is that focus state because of selection via mouse or touchpad is annoying and should appear only for keyboard navigation. I think that property is redundant - i'll implement keyboard-only focus state for all component (after discussion via https://github.com/epicmaxco/vuestic-ui/issues/2200#issuecomment-1206972780).

Reply:
@aluarius can't you explain to me why would items be focusable? (just checked a couple of examples and never that's the case)

Maybe as a prop at most?